### PR TITLE
Issue 175 Updates to presentation of Reserved_Keys chapter

### PR DESCRIPTION
**Pull request by dsolt**
Clearer presentation of realms and partitioning keys into realms.
Also changed some labels to be more consistent.

Signed-off-by: dsolt@us.ibm.com

### CODE REVIEW

**Comment by dsolt** [comment]
Please use emoji reactions ON THIS COMMENT to indicate your position on this proposal.

    You do not need to vote on every proposal
    If you have no opinion, don't vote - that is also useful data
    If you've already commented on this issue, please still vote so
    we know your current thoughts
    Not all proposals solve exactly the same problem, so we may end
    up accepting proposals that appear to have some overlap
    This is not a binding majority-rule vote, but it will be a very
    significant input into the corresponding ASC decision.

Here are the meanings for the emojis:

    Hooray or Rocket: I support this so strongly that I
    want to be an advocate for it
    Heart: I think this is an ideal solution
    Thumbs up: I'd be happy with this solution
    Confused: I'd rather we not do this, but I can tolerate it
    Thumbs down: I'd be actively unhappy, and may even consider
    other technologies instead
    If you want to explain in more detail, feel free to add another
    comment, but please also vote on this comment.

File: Chap_API_Reserved_Keys.tex [hunk 1]
```diff
@@ -377,11 +427,9 @@ \subsection{Process realm attributes}
 }
 %
 \declareAttribute{PMIX_PROCID}{"pmix.procid"}{pmix_proc_t}{
-Process identifier. Used as a key in \refapi{PMIx_Get} to retrieve the
-caller's own
-process identifier in a portion of the program that doesn't have access
-to the memory location in which it was originally stored (e.g., due to a
-call to \refapi{PMIx_Init}). The process identifier in the \refapi{PMIx_Get} call is ignored in this instance.
+The caller's process identifier.  
+The value returned is identical to what PMIx_Init or PMIx_Tool_init provides.
```
Comment (jjhursey) [inline Chap_API_Reserved_Keys.tex:431]: Editorial tex update.
```suggestion
The value returned is identical to what \refapi{PMIx_Init} or \refapi{PMIx_Tool_init} provides.
```

**Comment by dsolt** [comment]
[pmix-standard-reserved-keys.pdf](https://github.com/pmix/pmix-standard/files/8017366/pmix-standard-reserved-keys.pdf)

Here is a pdf showing the changes to this chapter with green highlighting for new text, red for changed text and grey for moved text.

File: Chap_API_Reserved_Keys.tex (hunk 1 above)
Comment (dsolt) [reply Chap_API_Reserved_Keys.tex:431]: done

**Review by jjhursey — COMMENTED** [review]
Notes from the reading at 1Q 2022 ASC meeting - Day 1 (stopped at section 6.1.3

File: Chap_API_Reserved_Keys.tex [hunk 2]
```diff
@@ -306,6 +326,24 @@ \subsection{Application realm attributes}
...
 %
+\declareAttribute{PMIX_APP_MAP_TYPE}{"pmix.apmap.type"}{char*}{
+Type of mapping used to layout the application (e.g., \code{cyclic}).
+}
+%
+\declareAttribute{PMIX_APP_MAP_REGEX}{"pmix.apmap.regex"}{char*}{
+Regular expression describing the result of the process mapping.
+}
+
+\vspace{\baselineskip}
```
Comment (abouteiller) [inline Chap_API_Reserved_Keys.tex:337]: During reading there was a missing vspace, but it appears to be done already

File: Chap_API_Reserved_Keys.tex [hunk 3]
```diff
@@ -306,6 +326,24 @@ \subsection{Application realm attributes}
...
+
+The following application-related keys default to the realms described in their descriptions but can be
+retrieved retrieved from the application realm by setting the \refattr{PMIX_APP_INFO} attribute in the \refarg{info} array passed to \refapi{PMIx_Get}:
```
Comment (naughtont3) [inline Chap_API_Reserved_Keys.tex:340]: "retrieved retrieved" typo

File: Chap_API_Reserved_Keys.tex [hunk 4]
```diff
@@ -353,7 +391,19 @@ \subsection{Process realm attributes}
 \declareAttribute{PMIX_RANK}{"pmix.rank"}{pmix_rank_t}{
 Process rank within the job, starting from zero.
 }
-%
+
+\declareAttribute{PMIX_NSPACE}{"pmix.nspace"}{char*}{
+Namespace of the job - may be a numerical value expressed as a string, but is often an
+alphanumeric string carrying information solely of use to the system. Required to be unique
+within the scope of the host environment.  One cannot retrieve the namespace of an arbitrary process
+since that would require already knowing the namespace of that process.  However, a processes own
```
Comment (naughtont3) [inline Chap_API_Reserved_Keys.tex:399]: think "a processes own" should be ...a process' own... (singular possessive )

File: Chap_API_Reserved_Keys.tex [hunk 5]
```diff
@@ -118,17 +118,14 @@ \subsection{Session realm attributes}
...
 %
 \declareAttribute{PMIX_TDIR_RMCLEAN}{"pmix.tdir.rmclean"}{bool}{
-Resource Manager will cleanup assigned temporary directory trees.
+The Resource Manager will remove any directories or files it creates in
+\refattr{PMIX_TMPDIR}.
 }
 %
 \declareAttributeNEW{PMIX_HOSTNAME_KEEP_FQDN}{"pmix.fqdn"}{bool}{
 \acp{FQDN} are being retained by the \ac{PMIx} library.
 }
```
[the remark this answers is not in the document]
Comment (dsolt) [reply Chap_API_Reserved_Keys.tex:128]: done

File: Chap_API_Reserved_Keys.tex (hunk 3 above)
Comment (dsolt) [reply Chap_API_Reserved_Keys.tex:340]: done

File: Chap_API_Reserved_Keys.tex (hunk 4 above)
Comment (dsolt) [reply Chap_API_Reserved_Keys.tex:399]: done

**Comment by jjhursey** [comment]
PMIx ASC 1Q 2022

 * Passed the first vote with revision exception: 10 yes / 1 no / 0 abstain
 * Failed the first vote without the revision exception: 8 yes / 3 no / 0 abstain
 * This is eligible for a second vote in the 2Q 2022 ASC quarterly meeting

**Comment by jjhursey** [comment]
PMIx ASC 2Q 2022
 * Passed the second vote: 11 yes / 0 no / 0 abstain
 * Will be merged into the next release

**Comment by jjhursey** [comment]
Rebased version in https://github.com/pmix/pmix-standard/pull/406

**Comment by jjhursey** [comment]
Closing in favor of PR #406 - which is the same content, but rebased onto the current `master`

**Pull request closed without merging**